tests/test_arm.c

```diff
@@ -248,7 +248,7 @@ static void test()
 		},
 	};
 
-	uint64_t address = 0x1000;
+	uint64_t address = 0x80001000;
```

Comment:
on the current code (without your patch), `00 f0 10 e8` is `blx    	#0x80001024`. i dont see how this pull req changes the output?